worldedit-sponge/build.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url = "http://files.minecraftforge.net/maven" }
+        maven { url = "https://oss.sonatype.org/content/repositories/snapshots/" }
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'
+    }
+}
+
+apply plugin: 'net.minecraftforge.gradle.forge'
```

Comment:
You should really use a Vanilla workspace here so it can also run on SpongeVanilla. Currently, probably 95% can already work on SpongeVanilla, and there is no point requiring SpongeForge for a Sponge implementation.